applications/ironside/se/update/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _ironside_se_update_application:
+
+IRONside SE Update application
+##############################
+
+This application is used to trigger an update of the IRONside SE firmware.
+The new firmware can be found in the release zip of the IRONside SE.
+Both recovery and main IRONside SE firmwares can be updated.
```

Comment:
```suggestion
The *IRONside SE Update* application updates the IRONside SE firmware on a SoC that already has IRONside SE installed.
It can update both the main image and the recovery image of IRONside SE using the IRONside SE firmware release ZIP file.
```